mex-{{ cookiecutter.project_name }}/pyproject.toml

```
@@ -127,5 +127,15 @@ known-first-party = ["mex", "tests"]
 convention = "google"
 
 [build-system]
-requires = ["pdm-backend==2.4.6"]
-build-backend = "pdm.backend"
+requires = ["hatchling>=1.28.0"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build.targets.wheel]
+packages = ["mex"]
+
+[tool.hatch.build.targets.sdist]
+include = [
+    "mex",
+    "README.md",
+    "pyproject.toml",
+]
```

```suggestion
packages = ["mex"]
```

this also worked for me and the .tar.gz had the same content in the end. can you verify?